Onion.Core/Models/Error.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Onion.Core.Models
+{
+    public class Error
```

Comment:
There's already a default error class in .net with a message I believe guys, what's the value in creating another?

Reply:
Yes, I agree. I'll change by LogInfo